### DISCUSSION TITLE
npm package @oryd/kratos-client does not have a Github repository

### DISCUSSION THREAD

**Discussion by aaronpcz** [ory/kratos#1238] [Q&A] [answered]
I'm interested in why the [@oryd/kratos-client](https://www.npmjs.com/package/@oryd/kratos-client) does not have an associated Github repository, to take advantage of the dependency bot.

I'm currently using the v0.5.4 release, which has a warning with the [axios dependency](https://npmjs.com/advisories/1594       ).

I imagine the repository doesn't exist as it looks like the @oryd/kratos-client is generated.  

However having a Github repository does provide a reliable place to report issues that is directly linked to the npm package. What is the best channel to flag or raise issues such as this?

**Comment by aeneasr** [comment, marked as answer]
Please use https://www.npmjs.com/package/@ory/kratos-client instead - we are planning a new release with fixed axios by EOM

**Reply by vinckr** [reply]
This should be resolved by now 👍
